tvtk/pyface/ui/wx/scene.py

```diff
@@ -681,9 +683,12 @@ def _do_idle(event, window=window):
     def _lift(self):
         """Lift the window to the top. Useful when saving screen to an
         image."""
-        if self.render_window.off_screen_rendering:
-            # Do nothing if off screen rendering is being used.
-            return
+        ## Even when off_screen_rendering is On, the image size would
+        ## be wrong if we don't lift the window because the OnSize
+        ## event would not be called anymore
+        # if self.render_window.off_screen_rendering:
+        #     # Do nothing if off screen rendering is being used.
+        #     return
```

Comment:
This changes fixes the size on Ubuntu but leads to wrong images on Mac OS.
I realise this PR #316 is now interwined with #317.